### PR TITLE
clarify valid verification document types to user

### PR DESCRIPTION
**Pull request by rahulgudim**
# PR Checklist

Please check if your PR fulfills the following requirements
- [X] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [X] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/n/projects/2640063/stories/186676926

# A brief description of the changes

Current behavior:
Does not provide clarification on what verification documents can a user upload.

New behavior:
Provides clarification on what verification documents can a user upload. This is approved by @khadijah

Screenshot 2023-12-14 at 12 51 06 PM

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: app/views/insured/families/verification/_documents_verification_list.en.html.erb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <% state_abbreviation = site_key.upcase %>
 
 <div>
+<h4><b>Acceptable file types:</b></h4>
```
Comment (polographer) [inline app/views/insured/families/verification/_documents_verification_list.en.html.erb:4]: all of this should be on the translation file, not hardcoded string

File: app/controllers/insured/verification_documents_controller.rb [hunk 2]
```diff
@@ -135,7 +135,7 @@ def validate_file_type
 
       next if VlpDocument::ALLOWED_MIME_TYPES.include?(mime_type) && file.size < max_file_size_in_bytes
       File.delete(file_path) if File.exist?(file_path)
-      flash[:error] = "Unable to upload file. Please upload a file in PNG, JPEG, or PDF format and ensure it's under #{doc_limit_mb}MB."
+      flash[:error] = "Unable to upload file. Acceptable file types - PDF, JPEG, PNG, and GIF and ensure it does not exceed #{doc_limit_mb} MB."
```
Comment (polographer) [inline app/controllers/insured/verification_documents_controller.rb:138]: all of this should be on the translation file, not hardcoded string

**Review by polographer — CHANGES_REQUESTED** [review]

File: app/controllers/insured/verification_documents_controller.rb (hunk 2 above)
Comment (rahulgudim) [reply app/controllers/insured/verification_documents_controller.rb]: moved to translations.

File: app/views/insured/families/verification/_documents_verification_list.en.html.erb (hunk 1 above)
Comment (rahulgudim) [reply app/views/insured/families/verification/_documents_verification_list.en.html.erb]: moved to translations.

**Review by RyanEddyIC — DISMISSED** [review]

**Review by polographer — APPROVED** [review]

**Review by RyanEddyIC — APPROVED** [review]

**Pull request merged**